### ISSUE TITLE
Places: contained in place for halls in buildings

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#199]
Consider changes to search as well to include nested halls.

**Comment by troughc** [comment]
@saumier do we still need this? Contained in places have been added to the UI

**Comment by saumier** [comment]
@troughc Yes. This is to enable a user to pick the right place. In my opinion, if they see the choices that are contained inside a place, then they can choose the most specific place which is always better.
